### ISSUE TITLE
Possibility to run the tool outside of electron.

### ISSUE THREAD

**Issue by karahan** [mockoon/mockoon#514]
**Is your feature request related to a problem? Please describe.**
I'd like to run the tool for a longer time

**Describe the solution you'd like**
Instead of running the cli or the desktop app, it would be a cool feature to run the server with a web UI.
So I can run a docker image and let it run on my local server.
This way I don't have to start Mockoon everytime I start my project it removes a thing to do after grabbing my cup of coffee in the morning :)

**Additional context**
Basically a possibility to run the tool without the use of electron

**Comment by 255kb** [comment]
While this is a interesting idea, this is not planned and probably won't be anytime soon. The amount of work is huge as it's basically recreating a new Electron like wrapper around the Angular web application and the server. There are already many more important things on the roadmap that need to be done before we tackle something like this. 
As I don't want to give false hope to other users, I will close this issue (for now at least).
